learning.py

```diff
@@ -618,38 +618,114 @@ def predict(example):
 
 # ______________________________________________________________________________
 
-# A decision list is implemented as a list of (test, value) pairs.
 
+class DLTest:
+    def __init__(self):
+        self.attr_values = {}
 
-def DecisionListLearner(dataset):
-    """[Figure 18.11]"""
+    def add(self, attr, value):
+        """add new attribute and value to the test"""
```

Comment:
Minor edit, but can you please capitalize the start of your sentences, for consistency's case?

Reply:
Sure!